scripts/install_and_disable_systemd

```diff
@@ -0,0 +1,125 @@
+#! /usr/bin/env bash
+
+function install_and_disable_systemd
+{
+    if [ -z "$1" ]
+    then
+        # shellcheck disable=SC1078,SC1079
+        echo """Usage: source install_and_disable_systemd PATH [NAMES]
+
+    Installs all services from the paths in PATH(colon seperated) and disables and removes
+    all services which matches the names in NAMES (colon seperated).
```

Comment:
What is the risk that someone accidentally removes all service files using this?

Reply:
I think the chances are low. I don't see people executing this as they don't know what is does.